### PR TITLE
E-invoicing: Fixes & improvements

### PR DESCRIPTION
**Pull request by beganovich**
This features few improvements & fixes for e-invoicing, including license handling, quota improvements & validation errors.

### CODE REVIEW

File: app/Http/Controllers/EInvoiceController.php [hunk 1]
```diff
@@ -120,14 +121,17 @@ public function quota(ShowQuotaRequest $request): \Illuminate\Http\Response
...
+            return response()->json(['message' => $response->json('message')], 422);
         }
 
         if ($response->getStatusCode() === 400) {
-            return response($response->body(), 400);
+            return response()->json(['message' => $response->json('message')], 400);
         }
 
-        return response()->noContent(500);
+        return response()->json([
```
Comment (turbo124) [inline app/Http/Controllers/EInvoiceController.php:133]: ->json() inside ->json() does that double encode?

File: app/Http/Requests/EInvoice/Peppol/StoreEntityRequest.php [hunk 2]
```diff
@@ -81,8 +81,8 @@ public function rules(): array
             'acts_as_receiver' => ['required', 'bool'],
             'acts_as_sender' => ['required', 'bool'],
             'tenant_id' => ['required'],
-            'classification' => ['required', 'string'],
-            'vat_number' => [Rule::requiredIf(fn() => $this->input('classification') !== 'individual')],
+            'classification' => ['required', 'in:business,individual'],
+            'vat_number' => [Rule::requiredIf(fn() => $this->input('classification') === 'business')],
```
Comment (turbo124) [inline app/Http/Requests/EInvoice/Peppol/StoreEntityRequest.php:85]: There are more classifications than just business / individual, the rule is

!= individual  classification must be business

**Review by turbo124 — COMMENTED** [review]
Please see comments.

File: app/Http/Controllers/EInvoiceController.php (hunk 1 above)
Comment (beganovich) [reply app/Http/Controllers/EInvoiceController.php:133]: No, I believe $response->json is to get json key value.

File: app/Http/Requests/EInvoice/Peppol/StoreEntityRequest.php (hunk 2 above)
Comment (beganovich) [reply app/Http/Requests/EInvoice/Peppol/StoreEntityRequest.php:85]: Got it :+1:

**Pull request merged**